src/resources/elements/primeDesignSystem/pbutton/pbutton.scss

```diff
@@ -30,6 +30,7 @@
   user-select: none;
   color: $Neutral04;
   white-space: nowrap;
+  gap: 12px;
```

Comment:
we're not using this feature in this PR.  Do we need these changes?

Reply:
yes. pbuttons don't add spaces for items inside it except when it's using the loading icon. In our case, we are using our own custom icons and thus they won't have any space between them and the text. 

With this change, any custom icon we will be using will have the needed space